packages/core/lib/commands/console.js

```diff
@@ -4,7 +4,7 @@ const command = {
     "Run a console with contract abstractions and commands available",
   builder: {},
   help: {
-    usage: "truffle console [--network <name>] [--verbose-rpc]",
```

Comment:
I think we want to keep `--network` everywhere it appears in the usage (because it will be so common)

Reply:
We discussed this offline. Network is added later in the help function! :)